Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -1589,6 +1609,9 @@ private void LogCommandLineTooLong(SandboxedProcessInfo info)
             // If a PipProcessError was logged, the pip cannot be marked as succeeded
             Contract.Assert(!standardOutHasBeenWrittenToLog || status != SandboxedProcessPipExecutionStatus.Succeeded);
 
+            Tracing.Logger.Log.LogDetoursDebugMessage(m_loggingContext, m_pip.SemiStableHash, m_pip.GetDescription(m_context),
+                $"Done with the rest of the crap in {DateTime.UtcNow.Subtract(start)}");

```

Comment:
Nit: other word maybe? 💃

Reply:
as I said, this wasn't meant to make it to a PR :)